host/windows/syscall.c

```diff
@@ -1426,6 +1426,14 @@ int oe_syscall_close_ocall(oe_host_fd_t fd)
     return ret;
 }
 
+// Unsupported yet.
```

Comment:
is there an Issue # open to track this?  Or will it be done in this PR?

Reply:
I'll create one. Probably will leave that to the Windows support (in a separate PR).